content/release-notes/2023-12-12-console.md

```diff
@@ -0,0 +1,32 @@
+---
+description: Reset branch from parent and other improvements
+---
+
+### Branch reset
+
+Neon is pleased to announce the latest branching feature to help improve your development workflows: **branch reset**.
+
+With one click, you can now update your working branch to the latest available schema and data from its parent branch, avoiding labor-intensive and error-prone manual updates or branch restorations. This action is availble from both the Neon Console and the Neon CLI. For details, see [Reset branch from parent](/docs/manage/branches#reset-a-branch-from-parent).
+
+![Reset branch from parent](/docs/relnotes/reset_from_parent.gif)
+
+### Set context for the Neon CLI
+
+Using a new `neonctl` command, `set-context`, you can now set a background context for your CLI sessions, letting you perform project or branch-specific actions without having to specify the project or branch id in every command. See [Neon CLI commands — set-context](docs/reference/cli-set-context) for more detail.
+
+### Support for timescaledb 2.13.0 for Postgres 16
+
+The `timescaledb` extension for scheduling stored procedures was updated to a newer version that includes support for Postgres 16.
+
+| Postgres extension           | Old version   | New version   |
```

Comment:
I might drop this table. timescaledb was never available on our PG16 before now, so there's no upgrade of old to new version.